pagopa/io-app
# chore: [IOBP-969] New payment checkout outcome mapping

This PR depends on 
## ⚠️ This PR depends on [#436](https://github.com/pagopa/io-dev-api-server/pull/436) -> Please review the dependent PR before reviewing this ⚠️


## Short description
This pull request includes updates related to handling different payment errors in the wallet feature

## List of changes proposed in this pull request
- Updated the `getPropsForOutcome` function to handle `INSUFFICIENT_AVAILABILITY_ERROR`, `CVV_ERROR`, and `PLAFOND_LIMIT_ERROR` outcomes.
- Updated the `@pagopa/io-app-design-system` dependency to version `^2.0.6`.

## How to test
- Try to pay something
- Verify whether the new outcomes align with the design specifications 

## Preview
https://github.com/user-attachments/assets/aebdb055-d5d9-4a0e-8592-ab830dce4f0d
